content/en/containers/guide/_index.md

```diff
@@ -16,6 +16,7 @@ disable_toc: true
     {{< nextlink href="/containers/guide/changing_container_registry" >}}Changing Your Container Registry{{< /nextlink >}}
     {{< nextlink href="/containers/guide/sync_container_images" >}}Synchronize Datadog's images with a private registry{{< /nextlink >}}
     {{< nextlink href="/containers/guide/how-to-import-datadog-resources-into-terraform/" >}}How to Import Datadog Resources into Terraform{{< /nextlink >}}
+    {{< nextlink href="/containers/guide/manage-datadogpodautocaler-with-terraform/" >}}Manage DatadogPodAutoscaler with Terraform{{< /nextlink >}}
```

Comment:
This should fix the current break in `link_checks_preview`:
```suggestion
    {{< nextlink href="/containers/guide/manage-datadogpodautoscaler-with-terraform/" >}}Manage DatadogPodAutoscaler with Terraform{{< /nextlink >}}
```
I'm going to go ahead and commit this to trigger CI again, while I review the rest of the PR.